scripts/test-infra/verify.sh

```diff
@@ -24,6 +24,12 @@ make helm-lint
 echo "Running unit tests"
 make test
 
+# Upload coverage report
+curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
+curl -Os https://uploader.codecov.io/latest/linux/codecov
+chmod +x codecov
```

Comment:
I'd suggest to move the codecov tool installation to the "Install deps" part above where other installation and setup steps are done, too